conf/puppet/hiera.yaml

```diff
@@ -1,14 +1,26 @@
 ---
-:backends: yaml
-:yaml:
-  :datadir: "/tmp/shinesolutions/packer-aem/hieradata"
-:hierarchy:
-  - local
-  - "%{::os.family}/%{::os.name}-%{::os.release.major}"
-  - "%{::os.family}/%{::os.name}"
-  - "%{::platform_type}"
-  - "%{::component}"
-  - common
-:logger: noop
-:merge_behavior: native
-:deep_merge_options: {}
+version: 5
+defaults:
+  datadir: "/tmp/shinesolutions/packer-aem/hieradata"
+  data_hash: yaml_data
+hierarchy:
+  - name: Hiera Data
+    paths: 
+      - aws.yaml
```

Comment:
Why are aws.yaml, docker.yaml, and author-publish-dispatcher.yaml specified here? Shouldn't they be loaded from `%{::platform_type}.yaml` and `%{::component}.yaml` ?

Reply:
Puppet fails to find values without the additional hieras being specifically specified in global hiera